molgenis-api-permissions/src/main/java/org/molgenis/api/permissions/rsql/PermissionsQuery.java

```diff
@@ -0,0 +1,58 @@
+package org.molgenis.api.permissions.rsql;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class PermissionsQuery {
```

Comment:
Can you use an Autovalue object here?

Reply:
Not easily, the RSQL visitor expects a instantion of the object on which to call the setters.